README.md

```diff
@@ -45,11 +45,11 @@ See **Installation Instructions** for each available [release](https://github.co
 
 > ### Latest releases
 > 
-> | cardano-wallet                                                                            | Jörmungandr                                                                          |
-> | ---                                                                                       | ---                                                                                  |
-> | [v2019-11-18](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2019-11-18) | [v0.7.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.7.0)         |
-> | [v2019-11-14](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2019-11-14) | [v0.7.0](https://github.com/input-output-hk/jormungandr/releases/tag/v0.7.0)         |
-> | [v2019-11-07](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2019-11-07) | [v0.7.0-rc7](https://github.com/input-output-hk/jormungandr/releases/tag/v0.7.0-rc7) |
+> | cardano-wallet                                                                            | Jörmungandr (compatible versions)                                                      |
+> | ---                                                                                       | ---                                                                                    |
+> | [v2019-12-09](https://github.com/input-output-hk/cardano-wallet/releases/tag/v2019-12-09) | [v0.8.0-rc10](https://github.com/input-output-hk/jormungandr/releases/tag/v0.8.0-rc10) |
```

Comment:
are we sure?

Reply:
Hence my comment on the PR :/ 
I've reverted to `rc8`, it's a safer assumption. rc10 doesn't mention any _breaking changes_ but we have so-to-speak, only tested with `rc8`.

Reply:
ok, missed the comment. Yeah, better revert to rc8.

Reply:
We can update if we manage to verify rc10...